browser-extension/utils.js

```diff
@@ -47,21 +47,23 @@ export const fetchTournesolApi = async (url, method, data) => {
 }
 
 export const addRateLater = async (video_id) => {
-  const resp = await fetchTournesolApi('video/', 'POST', {video_id: video_id})
-    .then(() =>
-      fetchTournesolApi(
-        'users/me/video_rate_later/',
-        'POST',
-        {video: {video_id: video_id}}
-      )
-    );
+  const addVideoResp = await fetchTournesolApi('video/', 'POST', {video_id: video_id});
+  if(!resp || resp.status !== 409){
```

Comment:
When logged in, I'm getting errors here, because `resp` is used before being defined. Did you mean `addVideoResp` (sidenote: I'd also prefer to use full names instead of abreviation `addVideoResponse`. I find that makes the code more readable)

```
Uncaught (in promise) ReferenceError: Cannot access 'resp' before initialization
```

Reply:
Should we use `.then()` here and `.catch()` to have the certitude that the promise has been fulfilled?

Reply:
I find it easier to go with `try...catch` and `await`. And I dislike that we use sometimes one sometimes the other in our codebase currently